Fulfillment/Samples/Fulfilled order items feed.json

```diff
@@ -46,7 +46,7 @@
     "orderIdentificationTypeId" : "SHOPIFY_ORD_ID",
     "thruDate" : null
   }, {
-    "idValue" : "NNUS#10055",
+    "idValue" : "SMUS#10055",
```

Comment:
name here is different than above in line 4

Reply:
changed